composer.json

```diff
@@ -125,6 +125,7 @@
             "@composer test -- tests/Integrations/Curl",
             "@composer test -- tests/Integrations/Memcached",
             "@composer test -- tests/Integrations/Mysqli",
+            "@composer test -- tests/Integrations/Mongo",
```

Comment:
Just noticed this one is out of alphabetical order too, but it's less important here.